gui/main.js

```diff
@@ -100,7 +100,8 @@ const configDefaultSettings =
                   "max_retry_attempts" : 5, \
                   "enable_auto_retry" : "False", \
                   "tx_delay" : 0, \
-                  "auto_start": 0 \
+                  "auto_start": 0, \
+                  "notification": 1 \
```

Comment:
Can we name this more self explaining like "enable_system_notification" or something like that?

Reply:
Updated to enable_sys_notification.